addon/sidebar.html

```diff
@@ -6,16 +6,20 @@
     <link rel="stylesheet" href="sidebar.css"/>
   </head>
   <body>
-    <div id="home-container">
-      <a class="feedback-button" href="">
-        <span class="feedback-button__copy">Feedback</span>
-        <div class="feedback-button__icon"></div>
-      </a>
-      <section class="onboarding-section">
-        <p>
-          Use the context (right-click) menu to open pages in this sidebar.
-        </p>
-      </section>
+    <div class="page" id="landing-panel">
+      <div class="graphic">
+        <div class="graphic__overlay">
+          <div class="graphic__clouds"></div>
+          <div class=graphic__browser></div>
+        </div>
+        <div class="graphic__shadow"></div>
+      </div>
+      <h1 class="title">Welcome to Side View</h1>
+      <h3 class="subtitle">A <a title=" Visit Test Pilot" href="https://testpilot.firefox.com">Firefox Test Pilot</a> Experiment</h3>
+      <a class="default-link" href="#" title="Watch Tutorial">Watch Tutorial</a>
+      <a class="default-link" href="#" title="Give Feedback">Give Feedback</a>
+    </div>
+    </div>
```

Comment:
Does this have an extra `</div>` tag? I think I counted 6 opening `<div ` but 7 closing `</div>`.